chains/orchestrator-relays/runtime/dancelight/src/tests/inbound_queue_tests/message_processor_tests.rs

```diff
@@ -24,7 +24,7 @@ use {
     sp_core::{H160, H256},
     sp_runtime::DispatchError,
     tp_bridge::symbiotic_message_processor::{
-        InboundCommand, Message, Payload, SymbioticMessageProcessor, MAGIC_BYTES,
+        InboundCommand, Message, Payload, SymbioticInboundMessageProcessor, MAGIC_BYTES,
```

Comment:
Maybe `SymbioticInboundMessageProcessorV1`?

Reply:
I can rename it yes. @ParthDesai do you agree? do u envision this to be in line with your changes?